sdk/personalizer/Azure.AI.Personalizer/src/Azure.AI.Personalizer.csproj

```diff
@@ -32,6 +32,8 @@
 
   <ItemGroup>
     <PackageReference Include="Azure.Core" />
+    <PackageReference Include="Microsoft.RL" VersionOverride="1.0.18120003-INTERNAL" />
+    <PackageReference Include="System.Text.Json" VersionOverride="6.0.1" />
```

Comment:
Is `System.Text.Json` necessary? I thought this was already included?

Reply:
The default is old 4.* version. But it won't support [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]. So I have to override the version to a higher one.